src/frontend/typer/ressurs.ts

```diff
@@ -5,6 +5,13 @@ export enum RessursStatus {
     IKKE_TILGANG = 'IKKE_TILGANG',
     SUKSESS = 'SUKSESS',
 }
+export type ApiRessurs<T> = {
```

Comment:
Vil det bli problematisk for axios og apicontext dersom dette legges inn under Ressurs?

Reply:
Eller kalle typen under (`Ressurs`) for noe mer beskrivende?
Og evt ha en type med alle `type Ressurs = ApiRessurs & DenAndreTypenRessurs`, hvis det er behov